### PR TITLE
Adding simple TOC (table of contents) and fixing language links 

### PR DESCRIPTION
**Pull request by IPvSean**
##### SUMMARY
this is to help navigation of workshop exercises for the RHEL workshop.  This is a first attempt to address the problem outlined in this issue https://github.com/ansible/workshops/issues/683

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
- exercises

##### ADDITIONAL INFORMATION
I really doubt this will be controversial!  Please make comments below if you highly disagree with changes here

### CODE REVIEW

**Comment by IPvSean** [comment]
tagging @goetzrieger and @liquidat since you did a lot of work on this workshop :)

File: provisioner/network_dec13.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
```
Comment (liquidat) [inline provisioner/network_dec13.yml]: @IPvSean What are all those provisioner files for? Looks like this was an accidental add?

Comment (IPvSean) [reply provisioner/network_dec13.yml]: yes accidental add, good catch

**Comment by IPvSean** [comment]
@liquidat can you make another pass

**Comment by liquidat** [comment]
@IPvSean Looks great - two comments though:

The main readme probably doesn't need that: it probably distracts more than it helps.
Main README TOC
So I would not show a TOC on the main page.

Also, why do you include the main headline always? I don't see value there, wouldn't it make more sense to only list the sub-headlines?
Screenshot from 2020-03-13 18-14-07
I would not add the main header to the TOC, just the "sub-headers".

Thoughts?

**Comment by IPvSean** [comment]
> So I would not show a TOC on the main page.

I was on the fence here, so this feedback helped.  I have made a commit if you could relook at it now. https://ipvsean.github.io/workshops/exercises/ansible_rhel/

**Pull request merged**